cmd/util.go

```diff
@@ -324,14 +329,15 @@ func getKubeVersionFromKubelet() *KubeVersion {
 	out, err := cmd.CombinedOutput()
 
 	if err != nil {
+		glog.V(2).Info("Failed to query kubelet")
 		glog.V(2).Info(err)
 	}
 
 	return getVersionFromKubeletOutput(string(out))
 }
 
 func getVersionFromKubectlOutput(s string) *KubeVersion {
-	glog.V(2).Info(s)
+	glog.V(2).Infof(fmt.Sprintf("Kubectl output:\n%s", s))
```

Comment:
```suggestion
	glog.V(2).Infof("Kubectl output: %s", s)
```

Feel free to leave the newline in if you think it's necessary, but the fmt.Sprintf is superfluous